### DISCUSSION TITLE
能否打包aria2管理页面AriaNg

### DISCUSSION THREAD

**Discussion by lcexc** [tw93/Pake#183] [Q&A] [answered]
项目地址: https://github.com/mayswind/AriaNg

本地使用的话,启动时需要同时启动后台aria2, 用Pake能实现吗?

**Comment by Tlntin** [comment, marked as answer]
理论上可以，AriaNg应该是静态html，arir2你本地启动就行。

**Reply by Tlntin** [reply]
最新dev分支已支持静态文件打包。
目前暂时只能源码编译，后续将合并到pake-cli命令行打包工具中。
![image](https://user-images.githubusercontent.com/28218658/209783134-331f86b4-f8cc-4b02-a42c-2bddc0362a92.png)

**Reply by Tlntin** [reply]
可以试试了。
目前应该基本Ok了。
使用方法：
1. 卸载当前版本。
```bash
npm uninstall pake-cli -g
```
2. 安装最新dev版本。
```bash
npm install https://github.com/tw93/Pake.git#dev -g
```
3. 打包当前目录的index.html文件（暂时只支持单个文件）。
```bash
pake ./index.html
```
更多使用说明可以看dev分支的[文档](https://github.com/tw93/Pake/tree/dev/bin#%E7%94%A8%E6%B3%95)。
